sdk/azcore/doc.go

```diff
@@ -6,7 +6,7 @@
 // license that can be found in the LICENSE file.
 
 /*
-Package azcore implements an HTTP request/response middleware pipeline.
+Package azcore implements an HTTP request/response middleware pipeline used by the Azure SDKs.
```

Comment:
"Azure SDK**s**" used to be a messaging bugbear--there's only one SDK to rule them all!--but maybe not anymore; this splits the difference:
```suggestion
Package azcore implements an HTTP request/response middleware pipeline used by Azure SDK clients.
```

Reply:
fixed